README.md

```diff
@@ -18,14 +18,30 @@ MailHog is an email testing tool for developers:
 * View messages in the web UI, or retrieve them with the JSON API
 * Optionally release messages to real SMTP servers for delivery
 
-### Getting started
-
-1. Either:
-  * [Download the latest release](/docs/RELEASES.md) of MailHog for your platform
-  * [Run it from Docker Hub](https://registry.hub.docker.com/u/mailhog/mailhog/) or using the provided [Dockerfile](Dockerfile)
-  * [Read the deployment guide](/docs/DEPLOY.md) for other deployment options
-  * Install it with Homebrew on MacOS - `brew update && brew install mailhog`
-2. [Configure MailHog](/docs/CONFIG.md), or use the default settings:
+### Installation
+
+#### Manual installation
+[Download the latest release for your platform](/docs/RELEASES.md). Then
+[read the deployment guide](/docs/DEPLOY.md) for deployment options.
+
+#### MacOS
+```
+brew update && brew install mailhog
+```
+
+Then, start MailHog by running `mailhog` in the command line.
+
+#### Debian / Ubuntu
+```
+sudo apt-get -y install golang-go
+go get github.com/mailhog/MailHog
+```
+
+Then, start MailHog by running `~/work/bin/MailHog` in the command line.
```

Comment:
Can you please update this to /path/to/MailHog? The explicit path may cause some confusion

Reply:
Yeah, for example for my Ubuntu the valid path is `~/go/bin/MailHog`. If it's different paths on different Debian/Ubuntu versions - then it would be good to also mention how to figure out this path, it took me a while to figure out that it should be `~/go/bin/MailHog`, and I bet many users won't figure out it at all.